packages/babel-preset-typescript/test/fixtures/opts/allExtensions-true-isTSX-true/options.json

```diff
@@ -0,0 +1,13 @@
+{
+  "BABEL_8_BREAKING": false,
+  "overrides": [
+    {
+      "include": "*.vue",
+      "sourceType": "module",
+      "presets": [
+        ["@babel/preset-react"],
+        ["@babel/preset-typescript", { "allExtensions": true, "isTSX": true }]
```

Comment:
The test case here is derived from `babel-preset-taro`'s usage: https://github.com/NervJS/taro/blob/2039dce209d12c7c05b3f4b07c70dd7e598fd02f/packages/babel-preset-taro/index.js#L78-L80